docs/direct-access-urls.md

```diff
@@ -96,14 +101,21 @@ _caching content store_), should have dedicated configuration options:
     - If not set, the default system-wide setting will be used
       (`system.directAccessUrl.maxExpiryTimeInSec`).
 
+>**Note:** `{$connectorName}` depends on Alfresco Cloud Connector. For S3 Connector it is 's3', for Azure Connector it is 'az'.
+
 >**Note:** Callers within the platform (i.e. Java interfaces) can either request a specific
 > expiry time or rely on the default.
 
 >**Note:** When multiple S3 buckets are used for storage in Alfresco, each S3 Content Store can
 > be configured with either the default (common) S3 Connector-specific properties (i.e.
-> `connector.s3.directAccessUrl.enabled` & Co) OR new separate properties could be defined for
-> each and every store (e.g. `connector.s3store1.directAccessUrl.enabled`,
-> `connector.s3store2.directAccessUrl.enabled`, etc.).
+> `connector.s3.directAccessUrl.enabled` & common/default settings) OR new separate properties could be defined for
+> each and every store (e.g. `connector.s3.store1.directAccessUrl.enabled`,
+> `connector.s3.store2.directAccessUrl.enabled`, etc.).
+>
+> For multiple Azure blob containers, each of them can also be configured similarly to S3 buckets (i.e.
+> `connector.azure.directAccessUrl.enabled` & common/default settings) OR new separate properties could be defined for
```

Comment:
Is "connector.azure.directAccessUrl.enabled" correct or should it be "connector.az.directAccessUrl.enabled" ?

Reply:
Thanks for catching that - change incoming.